FirstProject/assets/evaluation.less

```diff
@@ -0,0 +1,64 @@
+//Evaluation less file
+
+.row {
+
+  .navigation_bar {
+    border-bottom: 2px solid darkgrey;
+    padding-top: 12px;
+    padding-bottom: 14px;
+  }
+
+  .logo_image {
```

Comment:
This kind of nesting needs to go away. Do not use nesting at all. in any place. Use simple, single level classes.